tests/conftest.py

```diff
@@ -7,20 +7,51 @@
 import pytest
 
 
+@pytest.fixture
+def default_config() -> dict[str, str]:
+    """
+    Get the minimal default configuration for cutting a cookie in tests.
+
+    This is used if `generate_package` is called without arguments.
+    """
+    return {
+        "github_owner": "test-user",
+        "project_short_description": "description",
+        "project_name": "Cookiecutter Test",
+        "project_slug": "cookiecutter-test",
+    }
+
+
+@pytest.fixture
+def default_config_with(default_config: dict[str, str]) -> typing.Callable:
+    """Extend or modify the default configuration with one additional value."""
+
+    def _wrapped_with(key: str, value: str) -> dict[str, str]:
+        default_config[key] = value
+        return default_config
```

Comment:
```suggestion
    def _wrapped_with(**kwargs: str) -> dict[str, str]:
        return {**default_config, **kwargs}
```

This would both deal with Copilots comment around changes here also affecting `default_config` and allow multiple keys to be modified which is currently not needed but I guess future proofs us.

Reply:
Oooooh. That is a nice one-liner. 👨‍🍳 💋

Reply:
Actually I forgot that from Python 3.9 we can use the `|` operator which is probably more readable

```suggestion
    def _wrapped_with(**kwargs: str) -> dict[str, str]:
        return default_config | kwargs
```